module/VuFind/src/VuFind/Controller/Plugin/Favorites.php

```diff
@@ -5,7 +5,7 @@
  *
  * PHP version 8
  *
- * Copyright (C) Villanova University 2010.
+ * Copyright (C) Villanova University 2010 - 2023.
```

Comment:
Very minor style nitpick: we don't usually put spaces around the - in date ranges -- i.e. `2010-2023` instead of `2010 - 2023`.